recipes/pillow/test/test_pillow.py

```diff
@@ -13,7 +13,7 @@ def test_basic():
     img.save(out_file, "png")
     out_bytes = out_file.getvalue()
 
-    EXPECTED_LEN = 313772
+    EXPECTED_LEN = 619474
```

Comment:
I presume this means you've actually run the tests and found the test didn't work? FWIW - this change does make sense - the original test was the Lenna image, which I refuse to use because [Lenna herself has requested people stop using that photo](https://en.wikipedia.org/wiki/Lenna).

Reply:
Yep; got this value both on Linux and iOS.